.github/workflows/publish-auxiliary-packages.yml

```diff
@@ -58,3 +58,10 @@ jobs:
           npm config list
           echo "Publishing packages as $(npm whoami)"
           npm run publish-auxiliary
+
+      - name: "Publish tags"
+        run: |
+          npx lerna list -a --json | \
+            jq -r '.[] |  select(.name != "mongosh") | .name + "@" + .version' | \
```

Comment:
Is `mongosh` excluded here because it's not scoped with `@mongosh/`? Maybe add a comment or even a test somewhere that reminds people to update this line if that ever changes

Reply:
I am excluding it here as it is the publishing auxiliary packages which would mean it'd try to publish `mongosh@x.x.x` multiple times and make the tag meaningless.

Come to think of it, it'd be trivial to extract this to a function as well and make this distinction easier to reason about (and test/enforce) so I'll do that